pkg/distributor/distributor.go

```diff
@@ -301,7 +301,7 @@ func (d *Distributor) Push(ctx context.Context, req *logproto.PushRequest) (*log
 	validatedLineSize := 0
 	validatedLineCount := 0
 
-	var validationErr error
```

Comment:
On second thought regarding the use of MultiError vs GroupedError, the error messages for invalid labels and lines too big contain information about the stream and the line length respectively.

Therefore (and contrary to the test scenarios we have in place), the GroupedError will not be very helpful as in many cases the error content will be slightly different so grouping won't work. I see two alternatives:
- Make errors better groupable: E.g. not printing the line length for lines exceeding the limit.
- Use MultiError right away. My concern is that we may return **huge** errors.